src/app/pages/component-sidenav/component-nav.html

```diff
@@ -0,0 +1,23 @@
+<div class="docs-component-viewer-nav">
+  <div class="docs-component-viewer-nav-content">
+    <nav *ngFor="let category of docItems.getCategories((params | async)?.section); let last = last;">
+      <button (click)="toggleExpand(category.id)"
+              [attr.aria-label]="'Expand the ' + category.name"
```

Comment:
I tried this out with VoiceOver. The label would be easier to understand as:
category.name + ', section toggle`

Reply:
Done :)